lib/src/sql/value/serde/ser/distance/mod.rs

```diff
@@ -31,7 +31,6 @@ impl ser::Serializer for Serializer {
 		match variant {
 			"Euclidean" => Ok(Distance::Euclidean),
 			"Manhattan" => Ok(Distance::Manhattan),
-			"Cosine" => Ok(Distance::Cosine),
```

Comment:
Cosine is probably going to become "Angular". Will be implemented in an incoming PR.